libparsec/crates/platform_device_loader/tests/units/load.rs

```diff
@@ -5,7 +5,9 @@
 #![allow(clippy::unwrap_used)]
 
 use crate::{load_device, LoadDeviceError};
-use libparsec_tests_fixtures::prelude::*;
+use libparsec_tests_fixtures::prelude::{
```

Comment:
stick to `use libparsec_tests_fixtures::prelude::*;` plz

Reply:
done